.github/CODEOWNERS

```diff
@@ -0,0 +1,47 @@
+# Default code owners when not specified.  
+# We prefer to explicitly specify owners for all subdirectories.
+*                                warner, dckc, Chris-Hibbert, FUDCo, michaelfig
```

Comment:
My understanding is that this sets the _default_ reviewer for the various directories, and one goal here is to prevent PRs being created by external people which never get reviewed because the external person did not now who to add as a reviewer.  I don't think there is any other implicit duty that comes to you as an "owner" on this.

The * clause on line 3 should almost never apply, as the explicit settings lower in the file take precedence.  But if you want to be taken off the list, I'm happy to do that.

Reply:
I'd say the expectation is that whoever the robot picks as a reviewer should either do the review or pick a more appropriate reviewer. Anything is better than ignoring it, and any of us have more information about who the best reviewer should be than an outsider.

Reply:
The robot will pick just one? I thought it would pick all five, leading to anybody/somebody/nobody syndrome.

I guess I don't feel that strongly. Let's try it and see.

Reply:
I fixed the syntax problem in the file.  Please review again!